pwndbg/wrappers/readelf.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import pwndbg.wrappers
+
+cmd_name = "readelf"
+
+@pwndbg.wrappers.OnlyWithCommand(cmd_name)
+def get_jmpslots():
+    local_path = pwndbg.file.get_file(pwndbg.proc.exe)
+    cmd = [get_jmpslots.cmd_path, "-r", local_path]
+    readelf_out = pwndbg.wrappers.call_cmd(cmd)
+
+    return '\n'.join(filter(_extract_jumps, readelf_out.splitlines()))
```

Comment:
Probably we don't need to join that strings. We should just `return filter(...)`.

NOTE: `filter(...)` returns an object that is a generator in Python 3, so to be able to do more then just iterate once over it, a cast to `tuple` or `list` is needed.